decisions.md

```diff
@@ -2,23 +2,43 @@
 layout: default
 ---
 
-## The Good Docs Project Decisions
+# Decisions
 
-### Criteria for Decision Making
+This page describes the decision making processes used within The Good Docs Project.
+
+**Version:** 2.0
+
+**Last updated:** August 2020
+
+## Voting
 
 Decisions are based on voting using the following:
 
-    +1 : For
-    +0: Mildly for, but mostly indifferent
-    0: Indifferent
-    -0: Mildly against, but mostly indifferent
-    -1 : Strongly against. This is expected to be accompanied by a reason.
+*   +1 : For
+*   +0: Mildly for, but mostly indifferent
+*   0: Indifferent
+*   -0: Mildly against, but mostly indifferent
+*   -1 : Strongly against. This is expected to be accompanied by a reason.
+
+## Simple decisions
+
+For most decisions, a motion is raised within one of The Good Docs Project open forums (slack, email list, weekly meeting, etc.) and those present vote and decide. 
+
+## Significant decisions
+
+Should a decision:
+
+*   Be of interest to others not present,
+*   Be strategic or impact many within the project,
+*   Be contentious, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+
+then a motion should be raised as an email thread on our [general email list](https://thegooddocsproject.groups.io/g/main). The title should start: “Motion: …”
 
-For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. Should a decision:
+All community members are encouraged to vote to demonstrate community support, however contentious motions will only count votes from PSC members.
 
-    Be strategic or impact many within TheGoodDocsProject,
-    Or if there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+A motion is considered passed when there are more PSC +1 votes than -1 votes, and either all PSC members have voted, or 4 days have passed. In the case of a tie, the PSC chair shall have the deciding vote.
 
-Then the motion should be referred to the PSC.
+## Record decisions
 
-For motions presented to the PSC, a motion is considered passed when there are more PSC +1 votes than -1 votes, and either all PSC members have voted, or 4 days have passed. In the case of a tie, the PSC chair shall have 1.5 votes.
+*   All decisions should be captured in meeting minutes or forwarded to our email list. 
+*   Results of significant decisions should be copied into our [decision register](https://github.com/thegooddocsproject/governance/wiki/Decision-Log-2010).
```

Comment:
This URL seems to be incorrect.

Reply:
Ah yeah, it should be the year 2020, not 2010 (in the URL).

Reply:
fixed